tests/test_specific_rules.py

```diff
@@ -123,6 +124,8 @@ def test_new_terms_fields(self):
                 # checks if new terms field(s) are in ecs, beats non-ecs or integration schemas
                 queryvalidator = QueryValidator(rule.contents.data.query)
                 _, _, schema = queryvalidator.get_beats_schema([], beats_version, ecs_version)
+                for index_name in rule.contents.data.index:
+                    schema.update(**ecs.flatten(ecs.get_index_schema(index_name)))
```

Comment:
non-ecs schema was not being added originally, thus the specific AWS field was not found to be valid.